python/cli/cli.py

```diff
@@ -110,8 +110,10 @@ async def main():
     keys_file = keys_local if os.path.exists(keys_local) else keys_global
 
     # load the api keys and other settings from a JSON config
-    with open(keys_file, encoding="utf-8") as file:
-        keys = json.load(file)
+    keys = {}
+    if os.path.isfile(keys_file):
```

Comment:
We need an "else" statement here if we add a conditional.

Reply:
added 👌